File: PythonCode/1038.py
cod,qtde = input().split()
cod = int(cod)
qtde = int(qtde)
if cod == 1:
    valor = 4 * qtde
elif cod == 2:
    valor = 4.5 * qtde
elif cod == 3:
    valor = 5 * qtde
elif cod == 4:
    valor = 2 * qtde
elif cod == 5:
    valor = 1.5 * qtde
print("Total: R$ %.2f" %valor)

File: C++Code/1038.cpp
//Questão: Lanche
#include <iostream>
#include <iomanip>
 
 using namespace std;

 int main(){
     int cod, qtde;
     double valor;
     cin >> cod >> qtde;
      if (cod == 1){
        valor = 4 * qtde;
      }
      else if (cod == 2)
      {
          valor = 4.5 * qtde;
      }
      else if (cod == 3)
      {
          valor = 5 * qtde;
      }
      else if (cod == 4)
      {
          valor = 2 * qtde;
      }
      else if (cod == 5)
      {
          valor = 1.5 * qtde;
      }
      
      cout << fixed << setprecision(2) << "Total: R$ "<< valor << endl;
      
    return 0;
 }